core/src/utils/test/playwright.ts

```diff
@@ -106,6 +106,8 @@ export const test = base.extend<CustomFixtures>({
     page.setIonViewport = async () => {
       const currentViewport = page.viewportSize();
 
+      await page.waitForSelector('ion-content');
```

Comment:
With the other fix I merged into `main`, is this still needed?

Reply:
The behavior it was added to resolve, was fixed by your other changes. There is some value of it existing, in that we shouldn't be calling `setIonViewport` on a test template that doesn't use `ion-content`. If we use the `waitForSelector`, an invalid test will fail and help inform that that tag is expected in the test file. I think this is rare though, so I'd be happy to remove for now and re-add if we would find value in the safe guard.